hassio/src/addon-view/info/hassio-addon-info.ts

```diff
@@ -205,6 +210,18 @@ class HassioAddonInfo extends LitElement {
               `
             : ""}
           <div class="security">
+            <ha-label-badge
```

Comment:
I still think we should iconize it... 🤷‍♂️ 
As the text will never fit in the badge itself.

Reply:
Sure, we can do that, but I think that will be confusing.
I'll throw up some examples in a few mins.

Reply:
![image](https://user-images.githubusercontent.com/15093472/80798771-7199f000-8ba5-11ea-8b79-b3e216dd51c5.png)
![image](https://user-images.githubusercontent.com/15093472/80798774-7363b380-8ba5-11ea-8cce-9cd618ea23bc.png)
![image](https://user-images.githubusercontent.com/15093472/80798780-76f73a80-8ba5-11ea-9981-f65558e156fd.png)
![image](https://user-images.githubusercontent.com/15093472/80798803-824a6600-8ba5-11ea-9de3-731f20cd57b7.png)

Reply:
If they are added to the dialog as well, that might work:
![image](https://user-images.githubusercontent.com/15093472/80799185-7317e800-8ba6-11ea-9627-5253ca52512a.png)

Reply:
I like that approach by including the mapping in the description!

Reply:
Maybe we only have to show this if it isn't stable?

Reply:
I think they always should show, regardless of the stage.